evil-vars.el

```diff
@@ -969,6 +969,14 @@ These commands work better without this newline."
   :set 'evil-set-visual-newline-commands
   :initialize 'evil-custom-initialize-pending-reset)
 
+(defcustom evil-change-commands
```

Comment:
I'm not sure what the convention is (where to draw the line) between `defvar` and `defcustom`. If you think this shouldn't be a `defcustom`, let me know.

Reply:
I'd make it a variable because it's intended for programmatic use, not customization.  Other than that this looks good, thanks!